microsoftgraph/microsoft-graph-explorer-v4
# Fix: expand maxWidth allow more humanName chars

## Overview

There is issue #694 that was raised and Assigned in September. Since it has been a while since it was assigned and not done, I felt that I could provide a solution. 
- Fixes #694

### Demo

![Screenshot 2020-11-16 at 20 56 01](https://user-images.githubusercontent.com/48981919/99290571-82fbce80-284f-11eb-8218-ceeee4bce492.png)


### Notes

-> Simple way was to expand or allow stretching of maxWidth for `humanName column`. [Solution I went with], which seems to fix the horizontal scrollbar(I was unable to reproduce).
-> Another way was to make calculations based on the screen `window.innerWidth` and factor in the `25% that is max-width for the sidebar`. Somehow complex. 

## Testing Instructions

* Pull the latest changes in the branch
* Run the app and check on sidebar sections [human Name message] 
* Expected: it should not limit to 24 chars
